GpxLocationManager/Extensions/CLLocationDirection.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CLLocationDirection+Utility.swift
+//  Eunomia
+//
+//  Created by Ian Grossberg on 9/22/18.
+//  Copyright © 2018 Adorkable. All rights reserved.
+//
+
+import Foundation
```

Comment:
This line can be removed. If we were Java developers using IntelliJ, our IDE would tell us these things. 😀